cfg/2.0.0/definitions.yaml

```diff
@@ -399,7 +399,7 @@ groups:
             
     - id: 1.1.5
       description: "Ensure noexec option set on /tmp partition"
-      audit: "mount | grep /tmp"   
+      audit: "mount | grep -E '\\s/tmp\\s' | grep -v noexec"   
```

Comment:
Doesn't the -v mean this is excluding results that include `noexec`? 
I am not a fan of these test_items that compare an empty flag to an empty value...let's discuss a better way to achieve whatever it is supposed to achieve

Reply:
I don't like it either because there could be other errors or situation to get an empty output here, but those are part of the changes in the Linux CIS version 2.0.0
![image](https://user-images.githubusercontent.com/12915002/89125431-26b2b000-d4e7-11ea-8f3e-57c1cc99ac7b.png)